docs/cvl/lsp.md

```diff
@@ -0,0 +1,32 @@
+(lsp)=
+LSP Extension for VSCode
+========================
+
+If you're using VSCode to edit your CVL code, consider installing the
+[Certora Verification Language LSP extension](https://marketplace.visualstudio.com/items?itemName=Certora.evmspec-lsp).
+This extension contains several features to make editing CVL code easier,
+such as syntax highlighting, automatic syntax checking and auto-formatting.
```

Comment:
```suggestion
such as syntax highlighting, automatic syntax checking and formatting.
```

We hopefully don't auto-format, but trigger formatting only on users demand, correct?